src/CMakeLists.txt

```
@@ -45,30 +45,34 @@ if (NOT OGR2OGR)
 	message (WARNING "Cannot find program ogr2ogr - gmt will be unable to convert Shapefiles to GMT/OGR format")
 	set (GMT_CONFIG_OGR2OGR_MESSAGE "no" CACHE INTERNAL "OGR2OGR config message")
 else (NOT OGR2OGR)
-	set (GMT_CONFIG_OGR2OGR_MESSAGE "yes" CACHE INTERNAL "OGR2OGR config message")
+	execute_process(COMMAND bash "-c" "ogr2ogr --version | tr ',' ' ' | cut -f2 -d' '" OUTPUT_VARIABLE answer OUTPUT_STRIP_TRAILING_WHITESPACE)
```

```suggestion
	execute_process(COMMAND ${BASH} "-c" "ogr2ogr --version | tr ',' ' ' | cut -f2 -d' '" OUTPUT_VARIABLE answer OUTPUT_STRIP_TRAILING_WHITESPACE)
```

It's better to use `${BASH}` instead of `bash`.